libzen.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: libzen
```

Comment:
This includes header files:

https://apk.chaindag.dev/https/apk.cgr.dev/wolfi-presubmit/733c11d163a1493486f396aff2ee5c1d4a014cc1/x86_64/libzen-0.4.41-r0.apk@etag:e7e1936dae8c53fc0f9dfb7c71c2d721/usr/include/ZenLib/

Those should be in a `libzen-dev` package. See the `split/dev` pipeline. Packages that build against libzen should then depend on `libzen-dev` instead of `libzen`.